tensorflow/compiler/mlir/tosa/transforms/fuse_bias_tf.cc

```diff
@@ -68,32 +68,28 @@ LogicalResult ConvertTFBiasAddOp::matchAndRewrite(
 
   auto output_type =
       tf_biasadd_op.getResult().getType().dyn_cast<RankedTensorType>();
+
   // Not a ranked tensor output
   if (!output_type) return failure();
 
   auto value = tf_biasadd_op.value();
   auto bias = tf_biasadd_op.bias();
+  auto bias_shape = bias.getType().dyn_cast<RankedTensorType>().getShape();
```

Comment:
Just cast unless you check if null first before using.

Reply:
Got you. Thanks for letting me know.